# SARS-CoV-2_Bioinformatics


## 1. genome/genomics (Guangyuan)
### Data resources
- [EMBL-EBI: Covid-19 Data Portal](https://www.covid19dataportal.org/) The Covid-19 Data Portal developed and hosted by EMBL-EBI brings together relevant datasets for sharing and analysis in an effort to accelerate coronavirus research. It enables researchers to upload, access and analyse COVID-19 related reference data and specialist datasets as part of the wider European COVID-19 Data Platform. It includes some tools as well.
- [ViPR SARS-CoV-2 data portal | Virus Pathogen Resource](https://www.viprbrc.org/) The ViPR database integrates various types of data for multiple virus families. You can search the comprehensive database for sequences & strains, immune epitopes, 3D protein structures, host factor data, antiviral drugs, plasmid data. Further you can analyze the data online using sequence alignment, phylogenetic tree reconstruction, sequence variation (SNP), metadata-driven comparative analysis and BLAST. Visit the [SARS-CoV-2 data portal in ViPR](https://www.viprbrc.org/brc/home.spg?decorator=corona_ncov).
    > [B. E. Pickett, E. L. Sadat, Y. Zhang, J. M. Noronha, B. R. Squires, V. Hunt, M. Liu, S. Kumar, S. Zaremba, Z. Gu, L. Zhou, C. N. Larson, J. Dietrich, E. B. Klem, and R. H. Scheuermann, “ViPR: an open bioinformatics database and analysis resource for virology research,” Nucleic Acids Res, vol. 40, iss. D1, p. D593–D598, 2011.](http://dx.doi.org/10.1093/nar/gkr859)
- [GISAID (Global Influenza Surveillance AID)](https://platform.gisaid.org/epi3/frontend#24167b) International database of hCoV-19 genome sequences and related clinical and epidemiological data
    > [Y. Shu and J. McCauley, “GISAID: global initiative on sharing all influenza data – from vision to reality,” Eurosurveillance, vol. 22, iss. 13, 2017.](http://dx.doi.org/10.2807/1560-7917.es.2017.22.13.30494)
- [ENA (European Nucleotide Archive)](https://www.ebi.ac.uk/ena/pathogens/covid-19)
- [China National Center for Bioinformation (2019nCoVR)](https://bigd.big.ac.cn/ncov?lang=en) 国家生物信息中心NGDC 2019新型冠状病毒信息库
- [INB/ELIXIR-ES and TransBioNet: COVID-19 research](https://inb-elixir.es/news/inbelixir-es-nodes-and-transbionet-members-effort-covid-19-research)
- [GenBank Nucleotide Sequences](https://www.ncbi.nlm.nih.gov/nuccore/?term=txid2697049%5Borgn%5D) Provides rapid, open, and unrestricted access to virus nucleotide sequences and is the repository being recommended by NIAID and CDC for investigator and public health submissions. Due to the scale of data indexing, there may be a delay before new submissions are indexed and retrievable with a term-based query.
- [GenBank Protein Sequences](https://www.ncbi.nlm.nih.gov/protein/?term=txid2697049%5Borgn%5D) Provides rapid, open, and unrestricted access to virus conceptually translated protein sequences and is the repository being recommended by NIAID and CDC for investigator and public health submissions. Due to the scale of data indexing, there may be a delay before new submissions are indexed and retrievable with a term-based query.
- [NCBI Virus: SARS-CoV-2 data hub](https://www.ncbi.nlm.nih.gov/labs/virus/vssi/#/virus?SeqType_s=Nucleotide&VirusLineage_ss=Severe%20acute%20respiratory%20syndrome%20coronavirus%202,%20taxid:2697049) SARS-CoV-2 focused content from NCBI Virus, including links to related resources. Search, filter, and download the most up-to-date nucleotide and protein sequences from GenBank and RefSeq (taxid 2697049). Generate multiple sequence alignments and phylogenetic trees for sequences of interest. Provides one-click access to the Betacoronavirus BLAST database and relevant literature in PubMed.
- [Nextstrain COVID-19 genetic epidemiology](https://nextstrain.org/ncov) Open-source SARS-CoV-2 genome data and analytic and visualization tools
- [Sequence Read Archive (SRA)](https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?cmd=search_obj&m=&s=&term=txid2697049%5Borgn%5D&go=Search) Provides rapid, open, and unrestricted access to virus nucleotide or metagenomic sequence data and is the repository being recommended by NIAID and CDC for investigator and public health submissions. Due to the scale of data indexing, there may be a delay before new submissions are indexed and retrievable with a term-based query.
- [COVID-19 Genome Tracker](http://cov.borreliabase.org/)
- [ViralZone](https://viralzone.expasy.org/8996) SARS-CoV-2 protein seqs available at ViralZone
- [CoV-GLUE](http://cov-glue.cvr.gla.ac.uk/#/home) an online resource for comparative genomic analysis
- [Bio Basic](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=11&ved=2ahUKEwjG25utqbToAhWLXM0KHZH9B-Y4ChAWMAB6BAgFEAE&url=https%3A%2F%2Fwww.biobasic.com%2Fcoronavirus-covid-19-primers&usg=AOvVaw3Bs_HtOK_A69CqBBGRXrrz) Bio Basic is now providing ready-to-use primers and probes for COVID–19
- [Twist Bioscience](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&ved=2ahUKEwiPlbPpqrToAhWDQc0KHbrnBUQQFjAAegQIBxAB&url=https%3A%2F%2Fsynbiobeta.com%2Ftwist-bioscience-announces-availability-of-synthetic-sars-cov-2-rna-controls%2F&usg=AOvVaw0zSHAn2ik6WBaNxRaW6sU_) Twist Bioscience Announces Availability of Synthetic SARS-CoV-2 RNA Controls
- [Materials and Methods from Labome](https://www.labome.com/method/Current-PCR-Methods.html)
- [PubMed trending research papers (SARS-CoV-19)](https://www.ncbi.nlm.nih.gov/pubmed/trending/)
### Tools
- [PriSeT | Efficient De Novo Primer Discovery](https://github.com/mariehoffmann/PriSeT) Appropriate PCR primer pairs for DNA metabarcoding would match to a broad evolutionary range of taxa, such that we only need a few to achieve high taxonomic coverage. At the same time, the DNA barcodes between primer pairs should be different to allow us to distinguish between species to improve resolution. PriSeT finds a primer set P balancing both: high taxonomic coverage and high resolution. It is capable of processing large libraries and is robust against mislabeled or low quality references. It tackles the computationally expensive steps with linear runtime filters and efficient encodings.
PriSeT has been applied to 19 SARS-CoV-2 genomes and computed 114 new primer pairs with the additional constraint that the sequences have no co-occurrences in other taxa. These primer sets would be suitable for empirical testing.
    > [M. Hoffmann, M. T. Monaghan, and K. Reinert, “PriSeT: efficient de novo primer discovery,” bioRxiv, 2020.](http://dx.doi.org/10.1101/2020.04.06.027961)
- [CoVPipe | reference-based reconstruction of SARS-CoV-2 genomes](https://gitlab.com/rki_bioinformatics) CoVPipe is a highly optimized and fully automated workflow for the reference-based reconstruction of SARS-CoV-2 genomes based on next generation amplicon sequencing data using CleanPlex SARS-CoV-2 Panel (Paragon Genomics, Hayward, CA, USA) from swab samples. The pipeline is designed for reproducibility and scalability in order to ensure reliable and fast data analysis of SARS-CoV2 data.
- [poreCov](https://github.com/replikation/poreCov) The nanopore workflow poreCov carries out all necessary steps from basecalling to assembly depending on the user input, followed by lineage prediction of each genome using Pangolin. Furthermore, read coverage plots are provided for each genome to assess the amplification quality of the multiplex PCR. In addition, poreCov includes a quick time tree-based analysis of the inputs against reference sequences. poreCov is implemented in nextflow for full parallelization of the workload and stable sample processing.
- [V-Pipe | Mining viral genomes and improve clinical diagnostics](https://cbg-ethz.github.io/V-pipe/sars-cov-2/) V-Pipe has released a new version specifically adapted to analyze high-throughput sequencing data of SARS-CoV-2. It allows for the detection of within-host genetic variation of SARS-CoV-2 from viral NGS data.
    > [L. A. Carlisle, T. Turk, K. Kusejko, K. J. Metzner, C. Leemann, C. Schenkel, N. Bachmann, S. Posada, N. Beerenwinkel, J. Böni, S. Yerly, T. Klimkait, M. Perreau, D. L. Braun, A. Rauch, A. Calmy, M. Cavassini, M. Battegay, P. Vernazza, E. Bernasconi, H. F. Günthard, R. D. Kouyos, and Swiss HIV Cohort Study, “Viral diversity from next-generation sequencing of HIV-1 samples provides precise estimates of infection recency and time since infection.,” J Infect Dis, 2019.](http://dx.doi.org/10.1093/infdis/jiz094)
- [VIRify](https://github.com/EBI-Metagenomics/emg-viral-pipeline) VIRify can be used for the identification of coronaviruses in clinical and environmental samples. VIRify is a recently developed, generic pipeline for the detection, annotation, and taxonomic classification of viral and phage contigs in metagenomic and metatranscriptomic assemblies. VIRify’s taxonomic classification relies on the detection of taxon-specific profile hidden Markov models (HMMs), built upon a set of 22,014 orthologous protein domains and referred to as ViPhOGs. Included in this profile HMM database are 139 models that serve as specific markers for taxa within the Coronaviridae family.
- [VBRC Tools for Coronaviruses](https://4virology.net/organisms/coronaviridae/) The VBRC was developed for dsDNA viruses but has been adapted for coronaviruses. Only SARS-CoV-2 and closely related viruses will be added to this database. The VBRC provides unique tools that may be useful for the analysis of SARS-CoV-2.
- [VIRULIGN | Fast codon-correct alignment and annotation of viral genomes](https://github.com/rega-cev/virulign/tree/master/references/SARS-CoV-2) VIRULIGN is built for fast codon-correct alignments of large datasets, with standardized and formalized genome annotation and various alignment export formats. VIRULIGN has been adapted to SARS-CoV-2.
    > [P. J. K. Libin, K. Deforche, A. B. Abecasis, and K. Theys, “VIRULIGN: fast codon-correct alignment and annotation of viral genomes,” Bioinformatics, 2018.](http://dx.doi.org/10.1093/bioinformatics/bty851)
- [VIGOR4 | Viral Genome ORF Reader](https://github.com/JCVenterInstitute/VIGOR4) VIGOR4 (Viral Genome ORF Reader) is a Java application to predict protein sequences encoded in viral genomes. VIGOR4 determines the protein coding sequences by sequence similarity searching against curated viral protein databases. Vigor4 uses the VIGOR_DB project which currently has databases for the following viruses: Influenza (A & B for human, avian, and swine, and C for human), West Nile Virus, Zika Virus, Chikungunya Virus, Eastern Equine Encephalitis Virus, Respiratory Syncytial Virus, Rotavirus, Enterovirus, Lassa Mammarenavirus. SARS-CoV-2 release is coming (May, 1st).
    > [S. Wang, J. P. Sundaram, and D. Spiro, “VIGOR, an annotation program for small viral genomes,” BMC Bioinf, vol. 11, iss. 1, 2010.](http://dx.doi.org/10.1186/1471-2105-11-451)
- [Rfam COVID-19 Resources](https://rfam.org/covid-19) In response to the SARS-CoV-2 outbreak, Rfam produced a special release 14.2 that includes 10 new and 4 revised families that can be used to annotate the SARS-CoV-2 and other Coronavirus genomes with RNA families.
- [Covidex | Alignment-free machine learning subtyping for viral species](https://cacciabue.shinyapps.io/shiny2/) 
Covidex is an alignment-free machine learning subtyping tool for viral species, based on a random forest model trained over a kmer database. Currently, it supports FMDV and SARS-Cov-2 viral sequences. The tool allows a fast classification in pre-defined clusters (from the [Nextstrain database](https://nextstrain.org/)).
- [pangolin | Phylogenetic Assignment of Named Global Outbreak LINeages](https://pangolin.cog-uk.io/) Pangolin assigns a global lineage to query SARS-CoV-2 genomes by estimating the most likely placement within a phylogenetic tree of representative sequences from all currently defined global SARS-CoV-2 lineages based on the lineage nomenclature.
    > [A. Rambaut, E. C. Holmes, V. Hill, Á. O’Toole, J. McCrone, C. Ruis, L. du Plessis, and O. G. Pybus, “A dynamic nomenclature proposal for SARS-CoV-2 to assist genomic epidemiology,” bioRxiv, 2020.](http://dx.doi.org/10.1101/2020.04.17.046086)
- [BEAST 2 | Bayesian evolutionary analysis by sampling trees](https://www.beast2.org/) BEAST 2 is a cross-platform program for Bayesian phylogenetic analysis of molecular sequences. It estimates rooted, time-measured phylogenies using strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies but is also a framework for testing evolutionary hypotheses without conditioning on a single tree topology. BEAST 2 uses Markov chain Monte Carlo (MCMC) to average over tree space, so that each tree is weighted proportional to its posterior probability. BEAST 2 includes a graphical user-interface for setting up standard analyses and a suit of programs for analysing the results.
    > [R. Bouckaert, T. G. Vaughan, J. Barido-Sottani, S. Duchêne, M. Fourment, A. Gavryushkina, J. Heled, G. Jones, D. Kühnert, N. D. Maio, M. Matschiner, F. K. Mendes, N. F. Müller, H. A. Ogilvie, L. du Plessis, A. Popinga, A. Rambaut, D. Rasmussen, I. Siveroni, M. A. Suchard, C. Wu, D. Xie, C. Zhang, T. Stadler, and A. J. Drummond, “BEAST 2.5: an advanced software platform for Bayesian evolutionary analysis,” PLOS Comput Biol, vol. 15, iss. 4, p. e1006650, 2019.](http://dx.doi.org/10.1371/journal.pcbi.1006650)
- [Phylogeographic reconstruction using air transportation data](https://github.com/hzi-bifo/Phylogeography_Paper) Phylogeographic reconstruction using air transportation data can be used to study the global spread of the SARS-CoV-2 pandemic, especially in the early phases when air travel still substantially contributed to the spread of the virus. The method is currently adapted to consider both air travel and local movement data within countries during inference to reflect the changing worldwide movements in different phases of the pandemic.
    > [S. Reimering, S. Muñoz, and A. C. McHardy, “Phylogeographic reconstruction using air transportation data and its application to the 2009 H1N1 influenza A pandemic,” PLoS Comput Biol, vol. 16, iss. 2, p. e1007101, 2020.](http://dx.doi.org/10.1371/journal.pcbi.1007101)

## 2. related DNAses (Yongjing)

## 3. RNA (Youhuang)
### Resources
- [SARS-CoV-2 Sequencing Resources](https://github.com/CDCgov/SARS-CoV-2_Sequencing) a crowd-sourced collection of information, documentation, protocols and other resources for public health laboratories intending to sequence SARS-CoV-2 coronavirus samples.
- [SARS-COV-2 (COVID-19) BIOINFORMATICS RESOURCES](https://genexa.ch/sars2-bioinformatics-resources/) genexa provides resources useful for SARS-CoV-2 (COVID-19) genome index, unique kmers, and  complete genome assemblies.
- [Bioinformatics resources for SARS-CoV-2](http://www.clinbioinfosspa.es/CovidResources) lists bioinformatics resources useful track the evolution and progression of the SARS-CoV-2 virus as well as to manage genomics data of the virus
- [Coronavirus Teck Handbook](https://coronavirustechhandbook.com/) A collection of tools, websites and data relating to coronavirus.
### tools & databases
- [COVID-19 Data Portal](https://www.covid19dataportal.org/) EMBL-EBI will bring together COVID-19 datasets that have been submitted to its public databases, including ENA, UniProt, PDBe, EMDB, Expression Atlas and Europe PMC. The data, which have so far been collated, include genes, protein structures, electron microscopy data and scientific publications.
- [UK's SARS-CoV-2 genome sequencing consortium](https://www.cogconsortium.uk/data/)
- [COVID-19 literature review](https://covidreview.ca/) a shiny app provides higher-quality peer-reviewed current literature on COVID-19.
- [Genome Detective Coronavirus Typing Tool](https://www.genomedetective.com/app/typingtool/cov) a webserver to assemble all known virus genomes from next generation sequencing datasets
- [RNA Structural Covariation Above Phylogenetic Expectation](http://eddylab.org/R-scape/) R-scape looks for evidence of a conserved RNA structure by measuring pairwise covariations observed in an input multiple sequence alignment. It was applied to study the [evoluiton of long noncoding RNAs](https://www.nature.com/nmeth/journal/v14/n1/full/nmeth.4066.html).
- [RegRNA database](http://regrna2.mbc.nctu.edu.tw/). a regulatory RNA motifs identification tool.
- [BEAM web server](https://academic.oup.com/bioinformatics/article/34/6/1058/4583631)  a tool for structural RNA motif discovery.
- [RNA 3D Motif Atlas](http://rna.bgsu.edu/rna3dhub/motifs).
- [Rfam COVID-19 Resources](https://rfam.org/covid-19) Untranslated regions (UTR) are important functional elements that have conserved secondary structure and are responsible for multiple functions, including replication and packaging. [Blog](https://xfam.wordpress.com/tag/rfam/).
### RNA motifs
- [The architecture of SARS-CoV-2 transcriptome](https://www.cell.com/pb-assets/products/coronavirus/CELL_CELL-D-20-00765.pdf) a high-resolution map of the SARS-CoV-2 transcriptome and epitranscriptome.
At least 41 sites displayed substantial differences (over 20% frequency), indicating potential RNA modifications, by comparing the ionic current (called “squiggles”) between negative control and viral transcripts.  the most frequently observed motif is AAGAA. ‘AAGAA-like’ motif (including AAGAA and other A/G-rich sequences) are found throughout the viral genome but particularly enriched in genomic positions 28,500–29,500.
- [SECReTE cis-acting RNA elements](https://doi.org/10.1101/2020.04.20.050088) the SARS-CoV-2 genome contains 40 SECReTE (secretion-enhancing cis regulatory targeting element) motifs at an abundance of ~1.3 SECReTEs/kilobase (kb). The motif is "NYN" or "NNY" (where N is any nucleotide and Y = U or C). Mutation of SECReTE motif [enhances or inhibits mRNA stability and association with the ER](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1008248). This motif appears to promote mRNA stability, localization to ER, and translation.
- [ An in silico RNA folding map of SARS-CoV-2](https://doi.org/10.1101/2020.04.17.045161) The ScanFold program has been used to characterize its RNA folding landscape - highlighting regions of likely structure and function which serve as ideal targets for further analysis. The data is available [here](https://doi.org/10.1101/2020.04.17.045161).
- [RNA genome conservation and secondary structure in SARS-CoV-2 and SARS-related viruses](https://doi.org/10.1101/2020.03.27.012906)  Authors identified that 79 regions of length at least 15 nucleotides as exactly conserved over SARS-related complete genome. 106 ‘SARS-CoV-2-conserved-structured’ regions as potential targets for antivirals that bind to structured RNA.
- [Detection and sequence characterization of the 3'-end of coronavirus genomes harboring the highly conserved RNA motif s2m](https://eurekamag.com/research/052/515/052515898.php) A remarkably conserved 43-nucleotide-long motif present at the 3'-end of the genomes of several members of the polyadenylated RNA virus families Astroviridae, Coronaviridae, and Picornaviridae can be used for the detection and sequence characterization of the viruses harboring it. The mobile genetic element s2m is the consensus sequence [CGNGG(N)CCACGNNGNGT(N)ANNANCGAGGGT(N)ACAG](https://www.mdpi.com/2079-9721/4/3/27/htm). The function of s2m is unclear.
### others
- [A widespread Xrn1-resistant RNA motif composed of two short hairpins](https://doi.org/10.1101/522318) The 3’ untranslated region of several beny-and cucumovirus RNAs harbors a so-called ‘coremin’ motif that is required for Xrn1 stalling. the minimal benyvirus stalling site consists of two hairpins of 3 and 4 base pairs respectively. The 5’ proximal hairpin requires a YGAD (Y = U/C, D = G/A/U) consensus loop sequence, whereas the 3′ proximal hairpin loop sequence is variable. The sequence of the 9-nucleotide spacer that separates the hairpins is highly conserved and potentially involved in tertiary interactions. A role for Xrn1 and the host decay machinery has only been shown for the SARS coronavirus [nsp1](https://jvi.asm.org/content/jvi/86/17/9527.full.pdf). [Severe acute respiratory syndrome coronavirus nsp1 protein suppresses host gene expression by promoting host mRNA degradation](https://doi.org/10.1073/pnas.0603144103) Expression of nsp1, the most N-terminal gene 1 protein, prevented Sendai virus-induced endogenous IFN-beta mRNA accumulation without inhibiting dimerization of IFN regulatory factor 3, a protein that is essential for activation of the IFN-beta promoter.
- [a conserved BH3-like sequence](https://doi.org/10.1101/2020.04.09.033522) SARS-CoV E and SARS-CoV-2 E have a C-terminal BH3-like motif and a predicted interactome for E was identified.
## 4. interactions (Asif)
