# SARS-CoV-2_Bioinformatics


## 1. genome/genomics (Guangyuan)
### Data resources
- [EMBL-EBI: Covid-19 Data Portal](https://www.covid19dataportal.org/) The Covid-19 Data Portal developed and hosted by EMBL-EBI brings together relevant datasets for sharing and analysis in an effort to accelerate coronavirus research. It enables researchers to upload, access and analyse COVID-19 related reference data and specialist datasets as part of the wider European COVID-19 Data Platform. It includes some tools as well.
- [ViPR SARS-CoV-2 data portal | Virus Pathogen Resource](https://www.viprbrc.org/) The ViPR database integrates various types of data for multiple virus families. You can search the comprehensive database for sequences & strains, immune epitopes, 3D protein structures, host factor data, antiviral drugs, plasmid data. Further you can analyze the data online using sequence alignment, phylogenetic tree reconstruction, sequence variation (SNP), metadata-driven comparative analysis and BLAST. Visit the [SARS-CoV-2 data portal in ViPR](https://www.viprbrc.org/brc/home.spg?decorator=corona_ncov).
> [B. E. Pickett, E. L. Sadat, Y. Zhang, J. M. Noronha, B. R. Squires, V. Hunt, M. Liu, S. Kumar, S. Zaremba, Z. Gu, L. Zhou, C. N. Larson, J. Dietrich, E. B. Klem, and R. H. Scheuermann, “ViPR: an open bioinformatics database and analysis resource for virology research,” Nucleic Acids Res, vol. 40, iss. D1, p. D593–D598, 2011.](http://dx.doi.org/10.1093/nar/gkr859)


## 2. related DNAses (Yongjing)

## 3. RNA (Youhuang)
### Resources
- [SARS-CoV-2 Sequencing Resources](https://github.com/CDCgov/SARS-CoV-2_Sequencing) a crowd-sourced collection of information, documentation, protocols and other resources for public health laboratories intending to sequence SARS-CoV-2 coronavirus samples.
- [SARS-COV-2 (COVID-19) BIOINFORMATICS RESOURCES](https://genexa.ch/sars2-bioinformatics-resources/) genexa provides resources useful for SARS-CoV-2 (COVID-19) genome index, unique kmers, and  complete genome assemblies.
- [Bioinformatics resources for SARS-CoV-2](http://www.clinbioinfosspa.es/CovidResources) lists bioinformatics resources useful track the evolution and progression of the SARS-CoV-2 virus as well as to manage genomics data of the virus
- [Coronavirus Teck Handbook](https://coronavirustechhandbook.com/) A collection of tools, websites and data relating to coronavirus.
### tools & databases
- [COVID-19 Data Portal](https://www.covid19dataportal.org/) EMBL-EBI will bring together COVID-19 datasets that have been submitted to its public databases, including ENA, UniProt, PDBe, EMDB, Expression Atlas and Europe PMC. The data, which have so far been collated, include genes, protein structures, electron microscopy data and scientific publications.
- [UK's SARS-CoV-2 genome sequencing consortium](https://www.cogconsortium.uk/data/)
- [COVID-19 literature review](https://covidreview.ca/) a shiny app provides higher-quality peer-reviewed current literature on COVID-19.
- [Genome Detective Coronavirus Typing Tool](https://www.genomedetective.com/app/typingtool/cov) a webserver to assemble all known virus genomes from next generation sequencing datasets
- [RNA Structural Covariation Above Phylogenetic Expectation](http://eddylab.org/R-scape/) R-scape looks for evidence of a conserved RNA structure by measuring pairwise covariations observed in an input multiple sequence alignment. It was applied to study the [evoluiton of long noncoding RNAs](https://www.nature.com/nmeth/journal/v14/n1/full/nmeth.4066.html).
- [RegRNA database](http://regrna2.mbc.nctu.edu.tw/). a regulatory RNA motifs identification tool.
- [BEAM web server](https://academic.oup.com/bioinformatics/article/34/6/1058/4583631)  a tool for structural RNA motif discovery.
- [RNA 3D Motif Atlas](http://rna.bgsu.edu/rna3dhub/motifs).
- [Rfam COVID-19 Resources](https://rfam.org/covid-19) Untranslated regions (UTR) are important functional elements that have conserved secondary structure and are responsible for multiple functions, including replication and packaging. [Blog](https://xfam.wordpress.com/tag/rfam/).
### RNA motifs
- [The architecture of SARS-CoV-2 transcriptome](https://www.cell.com/pb-assets/products/coronavirus/CELL_CELL-D-20-00765.pdf) a high-resolution map of the SARS-CoV-2 transcriptome and epitranscriptome.
At least 41 sites displayed substantial differences (over 20% frequency), indicating potential RNA modifications, by comparing the ionic current (called “squiggles”) between negative control and viral transcripts.  the most frequently observed motif is AAGAA. ‘AAGAA-like’ motif (including AAGAA and other A/G-rich sequences) are found throughout the viral genome but particularly enriched in genomic positions 28,500–29,500.
- [SECReTE cis-acting RNA elements](https://doi.org/10.1101/2020.04.20.050088) the SARS-CoV-2 genome contains 40 SECReTE (secretion-enhancing cis regulatory targeting element) motifs at an abundance of ~1.3 SECReTEs/kilobase (kb). The motif is "NYN" or "NNY" (where N is any nucleotide and Y = U or C). Mutation of SECReTE motif [enhances or inhibits mRNA stability and association with the ER](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1008248). This motif appears to promote mRNA stability, localization to ER, and translation.
- [ An in silico RNA folding map of SARS-CoV-2](https://doi.org/10.1101/2020.04.17.045161) The ScanFold program has been used to characterize its RNA folding landscape - highlighting regions of likely structure and function which serve as ideal targets for further analysis. The data is available [here](https://doi.org/10.1101/2020.04.17.045161).
- [RNA genome conservation and secondary structure in SARS-CoV-2 and SARS-related viruses](https://doi.org/10.1101/2020.03.27.012906)  Authors identified that 79 regions of length at least 15 nucleotides as exactly conserved over SARS-related complete genome. 106 ‘SARS-CoV-2-conserved-structured’ regions as potential targets for antivirals that bind to structured RNA.
- [Detection and sequence characterization of the 3'-end of coronavirus genomes harboring the highly conserved RNA motif s2m](https://eurekamag.com/research/052/515/052515898.php) A remarkably conserved 43-nucleotide-long motif present at the 3'-end of the genomes of several members of the polyadenylated RNA virus families Astroviridae, Coronaviridae, and Picornaviridae can be used for the detection and sequence characterization of the viruses harboring it. The mobile genetic element s2m is the consensus sequence [CGNGG(N)CCACGNNGNGT(N)ANNANCGAGGGT(N)ACAG](https://www.mdpi.com/2079-9721/4/3/27/htm). The function of s2m is unclear.
### others
- [A widespread Xrn1-resistant RNA motif composed of two short hairpins](https://doi.org/10.1101/522318) The 3’ untranslated region of several beny-and cucumovirus RNAs harbors a so-called ‘coremin’ motif that is required for Xrn1 stalling. the minimal benyvirus stalling site consists of two hairpins of 3 and 4 base pairs respectively. The 5’ proximal hairpin requires a YGAD (Y = U/C, D = G/A/U) consensus loop sequence, whereas the 3′ proximal hairpin loop sequence is variable. The sequence of the 9-nucleotide spacer that separates the hairpins is highly conserved and potentially involved in tertiary interactions. A role for Xrn1 and the host decay machinery has only been shown for the SARS coronavirus [nsp1](https://jvi.asm.org/content/jvi/86/17/9527.full.pdf). [Severe acute respiratory syndrome coronavirus nsp1 protein suppresses host gene expression by promoting host mRNA degradation](https://doi.org/10.1073/pnas.0603144103) Expression of nsp1, the most N-terminal gene 1 protein, prevented Sendai virus-induced endogenous IFN-beta mRNA accumulation without inhibiting dimerization of IFN regulatory factor 3, a protein that is essential for activation of the IFN-beta promoter.
- [a conserved BH3-like sequence](https://doi.org/10.1101/2020.04.09.033522) SARS-CoV E and SARS-CoV-2 E have a C-terminal BH3-like motif and a predicted interactome for E was identified.
## 4. interactions (Asif)
