# SARS-CoV-2_Bioinformatics


## 1. genome/genomics (Guangyuan)

## 2. related DNAses (Yongjing)

## 3. RNA (Youhuang)
### tools & databases
- [COVID-19 Data Portal](https://www.covid19dataportal.org/) EMBL-EBI will bring together COVID-19 datasets that have been submitted to its public databases, including ENA, UniProt, PDBe, EMDB, Expression Atlas and Europe PMC. The data, which have so far been collated, include genes, protein structures, electron microscopy data and scientific publications.
- [COVID-19 literature review](https://covidreview.ca/) a shiny app provides higher-quality peer-reviewed current literature on COVID-19.
- [Genome Detective Coronavirus Typing Tool](https://www.genomedetective.com/app/typingtool/cov) a webserver to assemble all known virus genomes from next generation sequencing datasets
### RNA motifs
- [The architecture of SARS-CoV-2 transcriptome](https://www.cell.com/pb-assets/products/coronavirus/CELL_CELL-D-20-00765.pdf) a high-resolution map of the SARS-CoV-2 transcriptome and epitranscriptome.
At least 41 sites displayed substantial differences (over 20% frequency), indicating potential RNA modifications, by comparing the ionic current (called “squiggles”) between negative control and viral transcripts.  the most frequently observed motif is AAGAA. ‘AAGAA-like’ motif (including AAGAA and other A/G-rich sequences) are found throughout the viral genome but particularly enriched in genomic positions 28,500–29,500.
- [SECReTE cis-acting RNA elements](https://doi.org/10.1101/2020.04.20.050088) the SARS-CoV-2 genome contains 40 SECReTE (secretion-enhancing cis regulatory targeting element) motifs at an abundance of ~1.3 SECReTEs/kilobase (kb). The motif is "NYN" or "NNY" (where N is any nucleotide and Y = U or C). Mutation of SECReTE motif [enhances or inhibits mRNA stability and association with the ER](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1008248). This motif appears to promote mRNA stability, localization to ER, and translation.
- [ An in silico RNA folding map of SARS-CoV-2](https://doi.org/10.1101/2020.04.17.045161) The ScanFold program has been used to characterize its RNA folding landscape - highlighting regions of likely structure and function which serve as ideal targets for further analysis. The data is available [here](https://doi.org/10.1101/2020.04.17.045161).

## 4. interactions (Asif)
